install/operator/pkg/generator/assets/addons/publicApi/addon-public-oauthproxy.yml.tmpl

```diff
@@ -158,10 +79,11 @@ objects:
       spec:
         containers:
         - name: syndesis-public-oauthproxy
+          image: '{{ .Syndesis.Components.Oauth.Image }}'
           args:
             - --provider=openshift
-            - --client-id=system:serviceaccount:${OPENSHIFT_PROJECT}:syndesis-oauth-client
-            - --client-secret=${OPENSHIFT_OAUTH_CLIENT_SECRET}
+            - --client-id=system:serviceaccount:{{.OpenShiftProject}}:syndesis-oauth-client
+            - --client-secret=$(OPENSHIFT_OAUTH_CLIENT_SECRET)
```

Comment:
```suggestion
            - --client-secret=$(.OpenShiftOauthClientSecret)
```

Reply:
It's better to replace these with [environment variables support](https://github.com/openshift/oauth-proxy#environment-variables) in the oauth proxy, which avoids injecting secret values in deployment configs.